Sources/Parsing/Parsers/Float.swift

```diff
@@ -0,0 +1,172 @@
+extension BinaryFloatingPoint where Self: LosslessStringConvertible {
```

Comment:
I moved this file to be called "Float.swift" now that it's generic over a "float"-named protocol. Do we want to rename "Double.md" documentation, too? Or should we be consistent and just prefer "Double" everywhere?

Reply:
- [ ] Remove `Double.md` and mention that `Double` works in `Float.md`
- [ ] Mention that other integer types work in `Int.md`